ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/analytics/model/Constants.kt

```diff
@@ -1,3 +1,6 @@
 package org.cru.godtools.tutorial.analytics.model
 
 internal const val ADOBE_SITE_SECTION_TUTORIAL = "tutorial"
+internal const val ADOBE_SITE_SECTION_ON_BOARDING = "on_boarding"
+internal const val BASE_SCREEN_NAME_ON_BOARDING = "onboarding"
```

Comment:
I'm guessing the difference between the text of these 2 constants is intentional?